### DISCUSSION TITLE
How do I type 'o' in qwerty layout?

### DISCUSSION THREAD

**Discussion by nickp-real** [manna-harbour/miryoku#247] [Q&A] [answered]
Currently, I'm using BASE alpha with qwerty on Cantor Remix with colemak dhm on laptop.
The problem is when I type the key on the right pinky in the second column, I got ' (apostrophe).
I'm using num layout and left pinky to achieve 'o'.

Edit: https://github.com/manna-harbour/miryoku/discussions/85
This might be the answer, I will try.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/issues/35#issuecomment-1727468881.
